### ISSUE TITLE
implement support of meta transactions in dry-run on the node side

### ISSUE THREAD

**Issue by marc0olo** [aeternity/aeternity#3881]
The way to go is to implement support of meta transactions in dry-run on the node side

_Originally posted by @davidyuk in https://github.com/aeternity/aepp-sdk-js/issues/1401#issuecomment-1096724224_

**Comment by marc0olo** [comment]
it seems like we cannot handle errors occuring in a `GaMetaTx` on SDK side as long as there is no support for meta-transactions in dry-runs on the node side.

any chance to get this implemented?

**Comment by marc0olo** [comment]
@davidyuk there is reasoning behind not allowing to dry-run meta-transactions. it's about avoiding hijacking of transactions as it includes an already signed tx

**Comment by hanssv** [comment]
It doesn't really include an already signed Tx, the GAMetaTx (including the calldata for the auth-function) is _equivalent_ to a signed TX - so for symmetry we don't allow those to be dry-run.

**Comment by marc0olo** [comment]
Thanks for clarifying!

**Comment by davidyuk** [comment]
> symmetry

Do you mean for security? In general I got that, then do we have any other way to get the result of failed auth endpoint call? For example, if we used `require` with some message: https://github.com/aeternity/ga-multisig-contract/blob/9ddf0508bc46b512fe976c98a9796615729afcfb/contracts/SimpleGAMultiSig.aes#L102-L103

**Comment by marc0olo** [comment]
yeah, that's actually the main problem we face right now. I wanted to write tests to verify that these `require` statements are handled correctly
